deploy/scripts/src/accounts/test.ts

```diff
@@ -63,10 +63,10 @@ const profiles: ProfileList = {
       startRate: 1,
       timeUnit: '1s',
       preAllocatedVUs: 1,
-      maxVUs: 300,
+      maxVUs: 1000,
```

Comment:
```suggestion
      maxVUs: 300,
```
This should be kept at `300`.

Max expected journey duration is `30` seconds
Target load is `10 `requests per second target
∴ Max VUs should be `30 * 10 = 300`

Reply:
Updated in the next commit